hicommonwealth/commonwealth
# Remove unused dependencies

## Link to Issue
Closes: #6166

## Description of Changes
Cleans up the `package.json` and slightly reduces the slug size from 441 MB to 433 MB (8 MB reduction).

- Each commit lists a single removed package.
- Since this PR removes `crypto-js` the following vulnerability is resolved: https://github.com/hicommonwealth/commonwealth/security/dependabot/383

## Test Plan
- General click around:
  - Sign in with different wallets
  - Interact with various forum features
- Ensure the worker dynos function correctly on Heroku

## Deployment Plan

1. 

## Other Considerations

-
